vendor/assets/stylesheets/leaflet.css

```diff
@@ -246,7 +246,6 @@
 
 .leaflet-container {
 	background: #ddd;
-	outline: 0;
```

Comment:
I don't know if this is the best idea.. but it doesn't seem to change often and there's a bunch of tickets upstream about related things.

Reply:
Could we just override this in our styles? I could see us removing this vendor asset in favor of using the gem

Reply:
Not really; the best we can do override it with some other non-default outline which may or may not match the styles the browser actually uses.